configs/storybook/src/createConfig.ts

```diff
@@ -29,6 +29,9 @@ const createConfig = (): StorybookConfig => ({
     check: true,
   },
   staticDirs: ["../src/assets"],
+  docs: {
+    autodocs: true,
+  },
```

Comment:
This should probably be documented so consumers know their autodocs will be enabled by using this.